tests/SpecificationTest.php

```diff
@@ -2,12 +2,6 @@
 
 namespace Tanigami\Specification;
 
-use Doctrine\Common\Collections\Criteria;
```

Comment:
Those all were unused imports.

Reply:
@Bocmah 
Sorry for leaving you alone for so long without an answer.
The PR look OK to be merged. Thanks!